packages-content-model/roosterjs-content-model-plugins/lib/paste/WacComponents/processPastedContentWacComponents.ts

```diff
@@ -21,8 +23,16 @@ import type {
 const LIST_ELEMENT_TAGS = ['UL', 'OL', 'LI'];
 const LIST_ELEMENT_SELECTOR = LIST_ELEMENT_TAGS.join(',');
 
-const COMMENT_BG_COLOR_REST = 'rgba(209, 209, 209, 0.5)';
-const COMMENTS_TEXT_HIGHLIGHT_CLICKED = 'rgba(197, 139, 204, 0.5)';
+interface WacContext extends DomToModelContext {
```

Comment:
Instead of extend DomToModelContext, can we extend its member listFormat (type DomToModelListFormat)?

Reply:
Sure, updated to extend DomToModelListFormat instead in the latest iteration.